packages/hooks/src/useIsHasP2PSupportedCurrencies.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from 'react';
+import { useFetch, useInvalidateQuery } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+const useIsHasP2PSupportedCurrencies = () => {
+    const { client } = useStore();
+    const { account_list, is_authorize, loginid } = client;
+    const invalidate = useInvalidateQuery();
+    const { data, ...rest } = useFetch('website_status', { options: { enabled: is_authorize } });
+
+    // getting currency from the icon because account.currency is undefined
+    const real_account_currencies_list = account_list
+        .filter(account => !account.is_virtual)
+        .map(account => account.icon);
```

Comment:
@farhan-nurzi-deriv Maybe a bit out of scope, But you can steal [useAuthorize](https://github.com/binary-com/deriv-app/blob/51b3e49162c1e0cabdb290d75cf4be71fb3c0e41/packages/hooks/src/useAuthorize.ts) hook from [this PR](https://github.com/binary-com/deriv-app/pull/9018/files#diff-ea597938c31cef59e17aab2fb6a8632006eb4093e7ea92ffc664b491021e3bef) and use it in your hook so we don't rely on the client store data (less refactoring for later).

Feel free to ignore this comment 🙇🏻

Reply:
But the PR still open tho, should i copy the code from there 👀

Reply:
@farhan-nurzi-deriv You can get the file into your branch using git to keep the author's commit.
Anyway, You can ignore this comment and leave a todo for later to replace this with `useAuthorize` 🙇🏻